packages/perf-test/just.config.ts

```diff
@@ -11,6 +9,8 @@ import runPerfTest from './tasks/perf-test'
 // - Existing perf story format diverges from CSF format, requiring special loader.
 function bundleStories() {
   return async function() {
+    // delay require in case digest isn't built yet
```

Comment:
This will likely get fixed up when we split up the build steps a bit more per offline discussion